Prototope/TextLayer.swift

```diff
@@ -56,7 +95,31 @@ public class TextLayer: Layer {
 			updateSizePreservingOrigin() // Adjust width/height as necessary for new wrapping mode.
 		}
 	}
-
+	
+	public var textAlignment: Alignment {
+		get {
+			return Alignment(nsTextAlignment: label.textAlignment)
+		}
+		set {
+			label.textAlignment = newValue.toNSTextAlignment()
+			//No need to adjust size, since changing alignment doesn't influence it
+		}
+	}
+	
+	/** Distance from top of layer to the first line's baseline */
+	public var baselineHeight: Double {
+		return Double(label.font.ascender)
+	}
+	
+	/** Aligns this layer with the first baseline of the other layer */
+	public func alignWithBaselineOf(otherLayer: TextLayer) {
```

Comment:
Hm, this seems like a very common use-case (if not *the* most common use case for using the baseline). My guess was that @nsfmc was asking for a baseline getter (https://github.com/Khan/Prototope/pull/18#issuecomment-74729063) to do precisely this, so I’d be interested in hearing his feedback?

Also, it might make sense to rename this method to be more consistent with https://github.com/Khan/Prototope/blob/7f06c827c10fc36f233a3d791485c17644fd2331/Prototope/Layer%2BLayout.swift